tools/kagami/README.md

```diff
@@ -126,17 +127,23 @@ You can also choose output format:
 - Generate a genesis block in JSON format:
 
     ```bash
-    kagami genesis
+    kagami genesis --inlined-validator
```

Comment:
what happens if I provide both `--inlined-validator` and `--compiled-validator-path`?

Reply:
Fail with error. Clap support arg groups (accept only one argument).